README.md

```diff
@@ -180,10 +180,74 @@ curl -X 'GET' \
   --output ih9z9lwwv5r.zip
 ```
 
-# Manually uploading scans to Event Portal
+## Running a Discovery Scan and File Export Using the CLI
+
```

Comment:
@AndreaKelso Added additional context for the CLI to make it clear the CLI can be accessed from the docker container or the jar file.